src/main/java/uk/ac/ebi/eva/pipeline/io/readers/NonAnnotatedVariantsMongoReader.java

```diff
@@ -61,8 +62,10 @@ public NonAnnotatedVariantsMongoReader(MongoOperations template, String collecti
         if (studyId != null && !studyId.isEmpty()) {
             queryBuilder.add(STUDY_KEY, studyId);
         }
-        DBObject query = queryBuilder.add("annot.ct.so", new BasicDBObject("$exists", false)).get();
-        delegateReader.setQuery(query);
+        if (onlyNonAnnotated) {
+            queryBuilder.add("annot.ct.so", new BasicDBObject("$exists", false));
```

Comment:
We should try to avoid this if we already have defined those fields names in constants In the other PR.